distros/kubernetes/nvsentinel/charts/metadata-collector/templates/daemonset.yaml

```diff
@@ -50,7 +50,7 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default ((.Values.global).image).tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           args:
-            - --output-path={{ .Values.outputPath }}
+            - --output-path={{ .Values.outputPath | default .Values.global.metadataPath }}
```

Comment:
Can we only use only `.Values.global.metadataPath` only since we need it in multiple places?